### PR TITLE
Fixed buffer ownership flag in RasterBgra open

### PR DESCRIPTION
**Pull request by sebastianbaginski**
@marcusnaslund ready

### CODE REVIEW

File: source/draw/RasterBgra.ooc [hunk 1]
```diff
@@ -127,7 +127,8 @@ RasterBgra: class extends RasterPacked {
 		x, y, imageComponents: Int
 		requiredComponents := 4
 		data := StbImage load(filename, x&, y&, imageComponents&, requiredComponents)
-		This new(ByteBuffer new(data as UInt8*, x * y * requiredComponents), IntVector2D new(x, y))
+		buffer := ByteBuffer new(data as UInt8*, x * y * requiredComponents, true)
+		This new(buffer, IntVector2D new(x, y))
```
Comment (sebastianbaginski) [inline source/draw/RasterBgra.ooc]: @marcusnaslund : it crashed on your machine the last time we tried . I moved the buffer creation to separate line on purpose, let's see if this helps. I remember when moving initialization to separate line fixed invalid memory read related to Closure ...

Comment (marcusnaslund) [inline source/draw/RasterBgra.ooc]: Testing now. Cross your fingers.

Comment (marcusnaslund) [inline source/draw/RasterBgra.ooc]: *celebration*

Comment (sebastianbaginski) [inline source/draw/RasterBgra.ooc]: :dancers:

Comment (marcusnaslund) [inline source/draw/RasterBgra.ooc]: Hahah Github's emoticon support is a really useful feature.

**Pull request merged**